tribestream-api-registry-webapp/src/main/static/assets/templates/component_editable_number.jade

```diff
@@ -1,4 +1,5 @@
 div(tabindex=0)
+  span.empty(x-ng-if="!value") {{placeholder ? placeholder : Empty}}
```

Comment:
maybe you can change that to...

`span.empty(x-ng-if="!value && placeholder") {{placeholder}}`

because now it will try to use the vakue of the variable 'Empty'. I don't think we want to show the string 'Empty' if the placeholder value is not set.